src/gui/DatabaseWidget.cpp

```diff
@@ -617,23 +617,19 @@ void DatabaseWidget::openUrl()
 
 void DatabaseWidget::openUrlForEntry(Entry* entry)
 {
-    QString urlString = entry->resolveMultiplePlaceholders(entry->url());
-    if (urlString.isEmpty()) {
-        return;
-    }
-
-    if (urlString.startsWith("cmd://")) {
+    if (entry->url().startsWith("cmd://")) {
```

Comment:
We should call resolvePlaceholders before this check (like it was done before)

Example scenario:
The current entry has the url set to `{REF:A@I:<uuid>}` (a reference to another url field)
The other entry has the url field set to `cmd://firefox https://github.com`

In the current entry this check will be false (but should be true since the referred url does contains `cmd://`) and the url will be treated like a http url and the `github.com` link will be opened in the default browser (wrong behavior)

In the other entry the url will be correctly treated like a cmd and the execution alert below will be executed

Reply:
Done!